testsuite/tests/typing-modes/val_modalities.ml

```diff
@@ -461,7 +461,18 @@ Error: Signature mismatch:
          val f : int -> int @@ global many
        is not included in
          val f : int -> int @@ portable
-       The second is portable and the first is not.
+       The second is portable and the first is nonportable.
+|}]
+
+module F (M : sig val f : 'a -> 'a @@ global many end) = struct
```

Comment:
Were these not accepted before this patch?

Reply:
Yes - the first commit shows the pre-patch output.